lib/srv/app/common/header.go

```diff
@@ -40,7 +40,15 @@ const (
 	XForwardedSSL = "X-Forwarded-Ssl"
 
 	// TeleportAPIErrorHeader is Teleport-specific error header, optionally holding background error information.
-	TeleportAPIErrorHeader = "X-Teleport-API-Error"
+	TeleportAPIErrorHeader = "X-Teleport-Api-Error"
+
+	// TeleportAWSAssumedRole indicates that the incoming requests are signed
+	// with real AWS credentials of the specified assumed role by the AWS client.
+	TeleportAWSAssumedRole = "X-Teleport-Aws-Assumed-Role"
+
+	// TeleportAWSAssumedRoleAuthorization contains the original authorization
+	// header for requests signed by assumed roles.
+	TeleportAWSAssumedRoleAuthorization = "X-Teleport-Aws-Assumed-Role-Authorization"
```

Comment:
Can we add these to the `ReservedHeaders` below as well? And update `docs/pages/application-access/guides/connecting-apps.mdx` to include these headers under the part that says "The following headers are reserved and can't be rewritten"? Although as I am writing this I remember now that we don't do header rewriting for AWS apps, so maybe it doesn't matter, but better for future proofing....

Reply:
will do doc update separately.